### PR TITLE
docs: Flesh out readme

### PR DESCRIPTION
**Pull request by agardnerIT**
## This PR

Adds a little bit more code to the sample to make it more copy / paste friendly.

### CODE REVIEW

File: providers/openfeature-provider-flagd/README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ This provider is designed to use flagd's [evaluation protocol](https://github.co
 ## Installation
 
 ```
+pip install openfeature-sdk
```
Comment (federicobond) [inline providers/openfeature-provider-flagd/README.md:8]: We are going to assume in README files that users have already configured the SDK or know how to do so. This is not the best place for a tutorial on OpenFeature.

See the examples in the Java or JS SDK https://github.com/open-feature/java-sdk-contrib/tree/main/providers/flagd

File: providers/openfeature-provider-flagd/README.md [hunk 2]
```diff
@@ -14,7 +15,7 @@ Instantiate a new FlagdProvider instance and configure the OpenFeature SDK to us
 
 ```python
 from openfeature import api
-from openfeature.contrib.provider.flagd import FlagdProvider
```
Comment (federicobond) [inline providers/openfeature-provider-flagd/README.md:17]: Not sure why you deleted this import?

File: providers/openfeature-provider-flagd/README.md [hunk 3]
```diff
@@ -26,10 +27,17 @@ from openfeature import api
 from openfeature.contrib.provider.flagd import FlagdProvider
 from openfeature.contrib.provider.flagd.config import ResolverType
 
+
+
```
Comment (federicobond) [inline providers/openfeature-provider-flagd/README.md:31]: Please remove extra whitespace

File: providers/openfeature-provider-flagd/README.md [hunk 4]
```diff
@@ -26,10 +27,17 @@ from openfeature import api
...
 api.set_provider(FlagdProvider(
     resolver_type=ResolverType.IN_PROCESS,
     offline_flag_source_path="my-flag.json",
 ))
+
+openfeature_client = api.get_client()
+
+# Example: Retrieve a boolean flag
+flag_value = openfeature_client.get_boolean_value(flag_key="a-sample-flag", default_value=False)
```
Comment (federicobond) [inline providers/openfeature-provider-flagd/README.md:40]: Let's leave the details on using OpenFeature to the main SDK README and associated docs.

**Review by federicobond — CHANGES_REQUESTED** [review]

File: providers/openfeature-provider-flagd/README.md (hunk 1 above)
Comment (agardnerIT) [reply providers/openfeature-provider-flagd/README.md:8]: Hmm, I disagree. As an occasional user, I got here from a Google search and it didn't "just work". So off I go to Google and putting the pieces together myself, I eventually got it working. But it wasn't a good / fun UX.

**Comment by beeme1mr** [comment]
Hey @agardnerIT, would you be willing to add a [Python tutorial](https://github.com/open-feature/openfeature.dev/issues/537) to the docs? We could link to the tutorial from this repo once it's ready.

**Pull request closed without merging**